astroquery/alma/core.py

```diff
@@ -90,16 +100,12 @@ def query_region_async(self, coordinate, radius, cache=True, public=True,
         kwargs : dict
             Passed to `query_async`
         """
-        coordinate = commons.parse_coordinates(coordinate)
-        cstr = coordinate.fk5.to_string(style='hmsdms', sep=':')
-        rdc = "{cstr}, {rad}".format(cstr=cstr, rad=coord.Angle(radius).deg)
-
-        if payload is None:
+        coordinates = commons.parse_coordinates(coordinate)
+        if not payload:
             payload = {}
-        payload.update({'ra_dec': rdc})
-
+        payload['coordinates'] = coordinates
         return self.query_async(payload, cache=cache, public=public,
-                                science=science, **kwargs)
+                                science=science, ** kwargs)
```

Comment:
no space before kwargs
```suggestion
                                science=science, **kwargs)
```